### PR TITLE
Netfetch movies app

### PR DESCRIPTION
**Pull request by brahimdidi**
> ## Description
> > **Netfetch** is website made for users that are interested in rating movies It allows the visitor to browse an extensive collection of beautiful movies using the app. It also allows the user to like, comment and make a reservation for a given movie All data are saved thanks to the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270).
> 
> ### General requirements
> * [x]  There are no linter errors.
> * [x]  Followed the **Github flow**.
> * [x]  The documentation is professional.
> 
> ### HTML/CSS and JavaScript requirements
> * [x]  Followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
> * [x]  Followed the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)
> 
> ### Capstone project requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,96 @@
...
+
+# NetFetch
+
+> NetFetch is website for the latest and favorite TV shows, shipped with a really cool infinite scroll feature!. You can view the details of the show, "like" a show, and even comment. You can also see other comments which might give you a general idea about what people think about the show.
+> It is built with HTML, CSS and JavaScript with data from the TvMaze API(tv show data) and the Involvement API(likes and comments)
+
+## Video Presentation
+
+[Presentation Link]()
+
```
Comment (rotimiazeez) [inline README.md]: - [x] Kindly provide the link to the project presentation video 😉

File: src/__test__/commentCounter.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  test('Number of item in the arr', () => {
+    const arr = ['Hello people', 'Do you like me?', { go: 1, have: 101 }];
+    expect(counterComments(arr)).toBe(3);
+  });
+
+  test('Number of item in the arr', () => {
+    const arr = ['Soufiane', {}, [1, 'go'], 2022, 'JavaScript'];
+    expect(counterComments(arr)).toBe(5);
+  });
+});
```
Comment (rotimiazeez) [inline src/__test__/commentCounter.test.js:26]: - [x] 👏🏽 👏🏽 Good job implementing the tests.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM 👏🏽 , 
## Good job so far! 👍🏽 
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (vikipretium) [reply README.md]: Sure @rotimiazeez. I'll do it as soon as possible 👍🏼

Comment (Whoistolu) [reply README.md]: Great Presentation 👍🏽

File: src/index.html [hunk 3]
```diff
@@ -1,11 +1,285 @@
...
+
+    <!-- Button trigger modal -->
+    <!-- <button
+      type="button"
+      class="btn btn-primary"
+      data-bs-toggle="modal"
+      data-bs-target="#exampleModal"
+    >
+      Click me
+    </button> -->
```
Comment (Whoistolu) [inline src/index.html:57]: - [ ] [OPTIONAL ] Kindly remove all unused lines of code in order for us to have a clean code-base.

**Review by Whoistolu — APPROVED** [review]
### Hello Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**